dll/ntdll/rtl/libsupp.c

```diff
@@ -861,7 +863,8 @@ RtlDosApplyFileIsolationRedirection_Ustr(IN ULONG Flags,
         return STATUS_SXS_KEY_NOT_FOUND;
     }
 
-    if (NtCurrentPeb()->ProcessParameters &&
+    if ((Flags & RTL_DOS_APPLY_FILE_REDIRECTION_USTR_FLAG_RESPECT_DOT_LOCAL) &&
+        NtCurrentPeb()->ProcessParameters &&
```

Comment:
I wonder whether capturing the NtCurrentPeb's ProcessParameters into a "Peb" variable wouldn't simplify the code slightly more.

Reply:
A local var for one access is not simplifying imo.